src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -134,6 +134,7 @@ export default function FormNav(props) {
       current,
       formConfig.useCustomScrollAndFocus,
       index,
+      page.pageKey,
```

Comment:
focus would trigger on every chapter change, but not every page within chapter change, so adding this changes that.

Reply:
removed this part - turned out it wasn't necessary.